### PR TITLE
Release Develop

### PR DESCRIPTION
**Pull request by Allstreamer**
Since the baseline funtionality is now in place i think it's time to merge into main but i'm not sure whether to call it 1.0 or 0.2.

(Local versions are just to abitrary to add any sorting as of yet)

- [x] Move Unchanged to 1.0/0.2 in the CHANGELOG.md
- [x] Change Rust docs to use 1.0/0.2
- [x] Change README.md to use 1.0/0.2

### CODE REVIEW

**Comment by John15321** [comment]
just make sure you dont squash your develop branch, but rebase onto the master branch

File: Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 name = "pyver"
-description = "Python PEP-440 Version Parsing"
-authors = ["Allstreamer <allstreamer.contact@gmail.com>"]
+description = "Python PEP-440 Version Parser"
+authors = [
+    "Allstreamer <allstreamer.contact@gmail.com>",
+    "Jan Bronicki <janbronicki@gmail.com>",
+]
 license = "MIT"
 version = "0.1.0"
```
Comment (John15321) [inline Cargo.toml]: did you bump the crate version?

Comment (Allstreamer) [reply Cargo.toml]: No versions have been bumped yet still gotta do that but I ran out of time for today

Comment (John15321) [reply Cargo.toml]: ok

Comment (Allstreamer) [reply Cargo.toml]: Versions have been bumped @John15321

File: README.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+version numbers and for comparisons between
+[PEP-440](https://peps.python.org/pep-0440/) version numbers.
 
 ## Usage
 
 ```Toml
 [dependencies]
-pyver = "0.0.1"
+pyver = "1"
 ```
```
Comment (John15321) [inline README.md:18]: Did you add more examples to the README? Maybe just add a few more

Comment (Allstreamer) [reply README.md:18]: There are examples for most things the the rust docs, but you are probably right.

Comment (John15321) [reply README.md:18]: Then you can just add like 2 or 3 more and tell them to look into the cargo docs for more

Comment (Allstreamer) [reply README.md:18]: @John15321 2 examples have been added + links to crates.io and docs.rs

Comment (Allstreamer) [reply README.md:18]: Should be resolved

**Review by John15321 — APPROVED** [review]
Looks excellent! Good job!
Ship it 🚀

**Pull request merged**